views/account/generate_qr_code/index.ejs

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en" class="h-100">
+  <head>
+    <%- include('../../partials/header'); %>
+    <title>QR Code Generator</title>
+    <link rel="stylesheet" href="/account/generate_qr_code/style" />
+    <script src="/account/generate_qr_code/main" type="module"></script>
+    <script
+      src="https://cdnjs.cloudflare.com/ajax/libs/qrcodejs/1.0.0/qrcode.min.js"
+      integrity="sha512-CNgIRecGo7nphbeZ04Sc13ka07paqdeTu0WR1IM4kNcpmBAUSHSQX0FslNhTDadL4O5SAGapGt4FodqL8My0mA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    ></script>
+  </head>
+  <body class="h-100 text-center text-white bg-dark">
+    <div class="row col-12"><%- include('../../partials/navbar'); %></div>
+
+    <div
+      class="cover-container w-100 align-items-center h-75 p-3 mx-auto"
+    >
+      <div class="row col-12">
+        <main class="px-3">
+          <div class="form-check form-switch form-switch-xl">
+            <input class="form-check-input" type="checkbox" role="switch" id="instagram">
+            <label class="form-check-label" for="instagram">Instagram</label>
+          </div>
+          <div class="form-check form-switch form-switch-xl">
+            <input class="form-check-input" type="checkbox" role="switch" id="facebook">
+            <label class="form-check-label" for="facebook">Facebook</label>
+          </div>
+          <div class="form-check form-switch form-switch-xl">
+            <input class="form-check-input" type="checkbox" role="switch" id="twitter">
+            <label class="form-check-label" for="twitter">Twitter</label>
+          </div>
+
+          <form id="generate-form">
+            <input
+              id="url"
+              type="url"
+              placeholder="Enter a URL"
+              class="w-full border-2 border-gray-200 rounded p-3 text-grey-dark mr-2 focus:outline-none"
+              required
+            />
+
+            <select
+              class="w-full border-2 border-gray-200 rounded p-3 text-grey-dark mr-2 focus:outline-none"
+              name="size"
+              id="size"
+            >
+              <option value="100">100x100</option>
```

Comment:
I think we should fine-tune the number of options we give them for the QR code dimensions. I don't think the users are necessarily gonna use all of them

Reply:
Sounds good! Yeah we'll discuss the number of options when we all meet.

Reply:
It may even be a good idea to add this in the settings of the app: I think the size of stuff is probably a good addition to the accessibility settings.